docs/desktop-wpf/migration/differences-from-net-framework.md

```diff
@@ -49,9 +49,6 @@ Publicly defined CAS-related types were moved out of the WPF assemblies and into
 > - `PrivateAccessToTypeName`
 > - `AssemblyNameString`
 
-
-
 ## Next steps
 
 - [Learn how to port a .NET Framework WPF app to .NET Core.](convert-project-from-net-framework.md)
```

Comment:
Isn't there a lint rule for one blank line at the end of the file?

Reply:
This article is no longer part of this PR